save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -170,7 +170,7 @@ class DockerService(
             runCmd = listOf(
                 "sh", "-c",
                 "set -o xtrace" +
-                        " && curl -vvv -X POST \$${AgentEnvName.GET_AGENT_LINK.name} --output $SAVE_AGENT_EXECUTABLE_NAME" +
+                        " && curl -vvv -X POST \$${AgentEnvName.GET_AGENT_LINK.name}?agentId=\$${AgentEnvName.AGENT_ID.name} --output $SAVE_AGENT_EXECUTABLE_NAME" +
```

Comment:
unfortunately,  `AgentEnvName.AGENT_ID` is not available here.
potentially, we can create a `.run` as executable file which will be run. but this file should be runnable
it requires to mark copied file to docker as executable.

Reply:
reverted